sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-2022-2023.bpmn

```diff
@@ -841,17 +841,18 @@ dmnFullTimeIncomeThreshold</bpmn:text>
       <bpmn:extensionElements>
         <zeebe:ioMapping>
           <zeebe:output source="=if studentDataLivingathomeRent = null&#10;then &#34;no&#34;&#10;else&#10;  studentDataLivingathomeRent" target="studentDataLivingathomeRent" />
-          <zeebe:output source="=if studentDataTransportationCost = null&#10;then 0&#10;else&#10;  studentDataTransportationCost" target="studentDataTransportationCost" />
+          <zeebe:output source="=if studentDataReturnTripHomeCost = null&#10;then 0&#10;else&#10;  studentDataReturnTripHomeCost" target="studentDataReturnTripHomeCost" />
```

Comment:
Same question https://github.com/bcgov/SIMS/pull/4449/files#r1986186358

Reply:
As we talked `studentDataTransportationCost` was replaced to `studentDataReturnTripHomeCost`. I added a [comment ](https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/gh/bcgov/sims/4362#issuecomment-2711406960) to the ticket.